filters-uBO/extended_css_ELEMHIDE.txt

```diff
@@ -1,3 +1,4 @@
+hogaeng.co.kr#?#.bo_notice:has(.spon_b_icon1)
```

Comment:
Can be merged to `filters-share/extended_css_ELEMHIDE.txt`.
No such elements exist.